nuitka/nodes/NodeBases.py

```diff
@@ -45,8 +45,14 @@
 class NodeBase(NodeMetaClassBase):
     __slots__ = "parent", "source_ref"
 
+    # TODO: Come up to make lazy evaluation here
     # Avoid the attribute unless it's really necessary.
-    if Options.isFullCompat():
+    # from nuitka import Options
+    # if Options.isFullCompat():
+    #     __slots__ += ("effective_source_ref",)
+
+    # pylint: disable=W0125
+    if True:
```

Comment:
That seems wrong, I really want to have slots be dynamic or else memory usage is increased for a mode that is not used outside of Nuitka regression tests comparing to CPython.

Reply:
I've been forced to add that `pylint: disable=W0125` as that was making the linter complain and I wanted to push with the linter not giving any error/warning